data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleFactoryImpl.java

```diff
@@ -85,6 +85,8 @@
 
   private static final Logger logger = LoggerFactory.getLogger(ConsumerVerticleFactoryImpl.class);
 
+  private static final long DEFAULT_TIMEOUT_MS = 600_000L;
```

Comment:
would something like `TimeUnit.MILLISECONDS.convert(10L, TimeUnit.MINUTES)` be more readable?